app/controllers/api/v1/keywords_controller.rb

```diff
@@ -11,7 +11,11 @@ def index
       end
 
       def create
```

Comment:
I recommend to move the Job enqueuing to an ActiveRecord callback: That's a good use case to explore the [Active Record Callbacks guide](https://guides.rubyonrails.org/active_record_callbacks.html) 😎 

The idea is to let the controller simply "save" the keywords, without caring about anything else. 
The `model` will then know that each time a keyword is saved, it must "enqueue" the SearchKeyword job 😇 

This would also resolve your current todo of `enqueue "ALL" keywords) :)